packages/create-sitecore-jss/src/templates/react-native/ios/{{appName}}-tvOS/Info.plist

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file has pascalCaseName, and similar files, not sure how it will work if we will have lowercase register...again it means that it's required to do full testing

Reply:
Yes I noticed this, we will probably want to make these directory names pascal case if possible... I think it's required for the Java code to work.